src/components/toast-notifications/toastNotifications.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+angular.module('openshiftCommonUI')
+  .directive('toastNotifications', function(NotificationsService, $timeout) {
+    return {
+      restrict: 'E',
```

Comment:
Probably worth

```js
scope: {}
```

to create an isolated scope and avoid accidentally changing something in the parent scope